scripting/get5.sp

```diff
@@ -711,7 +712,8 @@ static Action Timer_InfoMessages(Handle timer) {
   }
 
   char readyCommandFormatted[64];
-  FormatChatCommand(readyCommandFormatted, sizeof(readyCommandFormatted), "!ready");
+  GetChatAliasForCommand(Get5ChatCommand_Ready, readyCommandFormatted, sizeof(readyCommandFormatted));
+  FormatChatCommand(readyCommandFormatted, sizeof(readyCommandFormatted), readyCommandFormatted);
```

Comment:
This is how it would be applied to all commands across the plugin for output in messages, I just only updated this "!ready" example for the PR so far.